CHANGELOG.md

```diff
@@ -1,3 +1,36 @@
+## 3.0.0
+February 28, 2019
```

Comment:
Is this realistic? Maybe put in March 1st or 4th as placeholder since we are still waiting for PRs

Reply:
@mikeng13 I think I saw that you were out next week. Do we feel comfortable releasing without you? Or should we release ahead of time even if the release then precedes the documented date?